packages/cli/generation/ir-migrations/src/migrations/v29-to-v28/__test__/migrateFromV29ToV28.test.ts

```diff
@@ -0,0 +1,15 @@
+import { AbsoluteFilePath, join, RelativeFilePath } from "@fern-api/fs-utils";
+import { createMigrationTester } from "../../../__test__/utils/createMigrationTester";
+import { V29_TO_V28_MIGRATION } from "../migrateFromV29ToV28";
+
+const runMigration = createMigrationTester(V29_TO_V28_MIGRATION);
+
+describe("migrateFromV29ToV28", () => {
+    it("snapshot", async () => {
+        const pathToFixture = join(AbsoluteFilePath.of(__dirname), RelativeFilePath.of("./fixtures/literal"));
+        const migrated = await runMigration({
+            pathToFixture,
+        });
+        expect(migrated).toMatchSnapshot();
```

Comment:
is there anything we want to assert about the migration (i.e. making sure that the referencedTypeIds for a particular type are correct)?